modules/server/src/test/scala/server/handlers/GrpcServerHandlerTests.scala

```diff
@@ -30,13 +30,13 @@ class GrpcServerHandlerTests extends RpcServerTestSuite {
 
   import implicits._
 
-  val handler: GrpcServerHandler[ConcurrentMonad] = new GrpcServerHandler[ConcurrentMonad]
+  val handler = GrpcServerHandler[ConcurrentMonad]
 
   "GrpcServer.Handler" should {
 
     "allow to start a GrpcServer" in {
 
-      runK(handler.start, serverMock).unsafeRunSync() shouldBe serverCopyMock
+      runK(handler.start, serverMock).unsafeRunSync() shouldBe (())
```

Comment:
If we're not checking the result, maybe we could just check that everything works fine with something like:
`runK(handler.start, serverMock).attempt.unsafeRunSync().isRight shouldBe true`